core/common/src/main/java/alluxio/cli/ValidationTool.java

```diff
@@ -21,5 +21,5 @@
    *
    * @return a json string of the test results
    */
-  String runTests();
+  String runTests() throws Exception;
```

Comment:
eg: InterruptedException which we cannot avoid

Reply:
because our server will runTests, i try to not affect our server, so all throwable will be transferred to failed task result. Not sure will the exception affects master rest service handler.

Reply:
I'll change `Exception` to `InterruptedException` then. I don't think the server should eat `InterruptedException`s.